Sources/ComposableArchitecture/Store.swift

```diff
@@ -144,8 +144,10 @@ public final class Store<State, Action> {
       if self.isSending {
         assertionFailure(
           """
-          The store was sent an action while it was already processing another action. This can \
-          happen for a few reasons:
+          The store was sent an action while it was already processing another action.
+          Sent action: \(debugCaseOutput(action))
+
+          This can happen for a few reasons:
```

Comment:
```suggestion
          The store was sent the action \(debugCaseOutput(action)) while it was already 
          processing another action.

          This can happen for a few reasons:
```

Reply:
I applied your suggestion 👍